internal/cmd/util/util.go

```diff
@@ -192,3 +192,9 @@ func ValidateRequiredFlags(flags *pflag.FlagSet, names ...string) error {
 	}
 	return nil
 }
+
+// WithGroup sets the GroupID field on the passed command.
+func WithGroup(groupId string, cmd *cobra.Command) *cobra.Command {
+	cmd.GroupID = groupId
+	return cmd
+}
```

Comment:
```suggestion
// WithGroup sets the GroupID field on the passed commands.
func WithGroup(groupId string, cmds ...*cobra.Command) []*cobra.Command {
	for i := range cmds {
		cmds[i].GroupID = groupId
	}
	return cmds
}
```

Maybe this reduces the duplication a bit? We can then unpack the array in the AddCommand function.

Reply:
Good idea! Maybe we can go a step further and also put the `cmd.AddGroup` call in there too.
Maybe something like this:

```go
// WithGroup creates a new group and adds the passed commands to it.
func WithGroup(cmd *cobra.Command, id string, title string, groupCmds ...*cobra.Command) {
	cmd.AddGroup(&cobra.Group{ID: id, Title: title})
	for _, groupCmd := range groupCmds {
		groupCmd.GroupID = id
	}
	cmd.AddCommand(groupCmds...)
}
```

What do you think about that?

Reply:
Done in bacaaf73d45314c931a185f5e1edba2981997c0e